src/Plugin/Block/GuidesContentsBlock.php

```diff
@@ -23,8 +21,9 @@ public function build() {
     $this->setPages();
     $links = [];
 
+    $options = $this->node->id() == $this->overview->id() ? ['attributes' => ['class' => 'active']] : [];
+    $links[] = $this->overview->toLink($this->overview->localgov_guides_section_title->value, 'canonical', $options);
     foreach ($this->guidePages as $guide_node) {
-      assert($guide_node instanceof NodeInterface);
```

Comment:
This was there as a type hind, equivalent of `/** @var \Drupal\node\NodeInterface $guide_node */` except actually PHP standard and correct - and if it explodes you know you've got something very broken.

Reply:
As I was already in the coding standards queue: https://www.drupal.org/project/coding_standards/issues/2425219#comment-12848176

Reply:
I've restores the missing NodeInterface assertion.